tools/Cron_jobs/mailwatch.cron.monthly

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# MailWatch for MailScanner
+# Copyright (C) 2003-2011  Steve Freegard (steve@freegard.name)
+# Copyright (C) 2011  Garrod Alwood (garrod.alwood@lorodoes.com)
+# Copyright (C) 2014-2017  MailWatch Team (https://github.com/mailwatch/1.2.0/graphs/contributors)
+
+# GeoIP Update
+test -x /usr/local/bin/mailwatch_geoip_update.php || exit 0
+/usr/local/bin/mailwatch_geoip_update.php  >/dev/null 2>&1
```

Comment:
Also, it may be wise to create a semi-random delay here.  Otherwise, it is possible for many instances of this to fire off at about the same time on multiple installs.  We don't want to accidentally ddos maxmind.